rollup.utils.js

```diff
@@ -44,8 +44,8 @@ function getBuildDirectories(packageName, folderName) {
   return { SOURCE_DIR, OUTPUT_DIR };
 }
 
-function createBanner(packageName, version, { executable = false } = {}) {
```

Comment:
Removing this since the `bin.js` entry points manually contain the Node shebang.